angular.json

```diff
@@ -123,6 +123,10 @@
           }
         }
       }
-    }},
-  "defaultProject": "web-app"
-}
+    }
+  },
+  "defaultProject": "web-app",
+  "cli": {
+    "analytics": false
+  }
```

Comment:
What does this do?

Reply:
Its the Angular cli feedback I chose no feedback to angular when creating a component.

Reply:
[Angular cli analytics docs](https://angular.io/cli/analytics)